src/store/Boards.ts

```diff
@@ -43,10 +44,14 @@ export interface BoardDescriptor extends BaseBoardDescriptor {
 	supportsBME280: boolean;
 }
 
+export type BoardTypes = BoardType | STMBoardTypes
```

Comment:
Is it really necessary to export both?

Reply:
For `STMBoardTypes`, I added a property `stm` to store the info needed and related to each board.
The easiest way I found was to add these boards to the `Boards` descriptor and by defining `BoardTypes` as this.

I would like to have an independant descriptor record for the STMboards but I didn't found an easy way to do this (and without needing to modify many files ...).
That's actually why I created this PR : to discuss about this solution.